README.md

```diff
@@ -43,6 +43,111 @@ The `pages/api` directory is mapped to `/api/*`. Files in this directory are tre
 
 This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
 
+## Getting Started With Docker
+First, install ["Docker"](https://www.docker.com/) on your local machine. 
+
+After installing docker, cd into the app directory and run the following commands:
+
+```bash 
+docker-compose build # this will build the application and install dependencies
+```
+
+```bash 
+docker-compose up # this will run the application
+```
+
+After running the above commands, the application should run on http://localhost:3000/
+
+To stop the application, open a new terminal window and run: 
+
+```bash 
+docker-compose down # this will stop the application
+```
+
+
+## Docker
+
+_Note_: This application is using Docker only for production builds and not for local development. For local development, the `npm run dev` command is the way to go.
+
+### Building Docker Images
+
+To build a Docker image for this application, run:
+
+```
+$ docker build -t digital-collections .
+```
+
+This command will build an image tagged (`-t`) as `digital-collections-app` using the current directory. Any changes to the application will require a new tagged image to be created to view those changes. Either remove the existing image (copy the image ID to use in the `docker image rm` command) and run the same command above:
```

Comment:
Minor: the examples use `digital-collections` without the `-app` at the end so remove it here to be consistent. Or you can add `-app` to the end of the other places where the `digital-collections` string is used for the image name.

Reply:
oy, good catch

Reply:
do you have a preference?

Reply:
I don't.